### DISCUSSION TITLE
求解github怎么换头像

### DISCUSSION THREAD

**Discussion by 0wuming0** [share121/inter-knot#2567] [其他]
我试了很多图片都说不能用
![微信图片_20240817193913](https://github.com/user-attachments/assets/8b738832-4d06-439b-8143-1dd6c250f783)
![微信图片_20240817195639](https://github.com/user-attachments/assets/965f0e9f-de37-418b-94c8-5ac1fcb3896d)
![微信图片_20240817195639](https://github.com/user-attachments/assets/92572d5a-d24f-4d31-9653-c7a3cf09fab5)

**Comment by BHGD-031212** [comment]
看看图片大小，<img src="https://bbs-static.miyoushe.com/static/2023/04/28/ad3e4d3d37302d240339f611b2458ef9_7921317972787298019.png" alt="米仔_比心" width="75" >只能用小于1Mb的图片

**Reply by 0wuming0** [reply]
是吗，我重新压缩看看

**Comment by 00zzxx** [comment]
图片太大了吧，或者格式不对？<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Reply by 0wuming0** [reply]
格式都是jpg，应该是太大了

**Comment by 0wuming0** [comment]
有热心绳网用户说太大了，我看了一下才45kb【悲】

**Comment by JobNumber-9527** [comment]
上传头像要把你的watt加速器关了才行

**Reply by 0wuming0** [reply]
成了【乐】
